From eda1fe83ba7605e62affec82d268cca97a90ff49 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Jan=C3=9Fen?= <janssen@mpie.de>
Date: Mon, 3 Mar 2025 13:55:48 +0100
Subject: [PATCH] grace-tensorpotential

---
 src/pyace/fitadapter.py | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/src/pyace/fitadapter.py b/src/pyace/fitadapter.py
index 58b930a..7fd6546 100644
--- a/src/pyace/fitadapter.py
+++ b/src/pyace/fitadapter.py
@@ -168,11 +168,11 @@ def setup_tensorpot(self, bbasisconfig: BBasisConfiguration, dataframe: pd.DataF
                         loss_spec: LossFunctionSpecification,
                         trainable_parameters_dict: Dict
                         ) -> BBasisConfiguration:
-        from tensorpotential.potentials.ace import ACE
-        from tensorpotential.tensorpot import TensorPotential
-        from tensorpotential.fit import FitTensorPotential
-        from tensorpotential.utils.utilities import batching_data, init_gpu_config
-        from tensorpotential.constants import (LOSS_TYPE, LOSS_FORCE_FACTOR, LOSS_ENERGY_FACTOR, L1_REG,
+        from tensorpotential.compat.pace.potentials.ace import ACE
+        from tensorpotential.compat.pace.tensorpot import TensorPotential
+        from tensorpotential.compat.pace.fit import FitTensorPotential
+        from tensorpotential.compat.pace.utils.utilities import batching_data, init_gpu_config
+        from tensorpotential.compat.pace.constants import (LOSS_TYPE, LOSS_FORCE_FACTOR, LOSS_ENERGY_FACTOR, L1_REG,
                                                L2_REG, AUX_LOSS_FACTOR)
         with warnings.catch_warnings():
             warnings.filterwarnings("ignore", category=UserWarning)
@@ -303,9 +303,9 @@ def setup_backend_for_predict(self, bbasisconfig):
             raise ValueError("`bbasisconfig` couldn't be None for FitAdapter.setup_backend_for_predict")
         log.info("Setting {} backend for predicting".format(self.backend_config.evaluator_name))
         if self.backend_config.evaluator_name == TENSORPOT_EVAL:
-            from tensorpotential.potentials.ace import ACE
-            from tensorpotential.tensorpot import TensorPotential
-            from tensorpotential.fit import FitTensorPotential
+            from tensorpotential.compat.pace.potentials.ace import ACE
+            from tensorpotential.compat.pace.tensorpot import TensorPotential
+            from tensorpotential.compat.pace.fit import FitTensorPotential
             ace_pot = ACE(bbasisconfig)
             tp = TensorPotential(ace_pot)
             self.fitter = FitTensorPotential(tensorpot=tp, eager=True)
