From b30bf2000b111e8a5b5e323fd518bda49a3fa58f Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Sun, 14 Jul 2024 09:26:32 +0200
Subject: [PATCH] Update CMakeLists.txt

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6bfcc23..a71092f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -55,7 +55,7 @@ set(CNPY_PATH lib/ace/utils/cnpy)
 set(CNPY_INCLUDE_PATH lib/ace/utils/)
 include_directories(${CNPY_INCLUDE_PATH})
 
-add_subdirectory(lib/pybind11)
+find_package(pybind11)
 
 
 # for advanced functionality
