From 3ec2e4079947bd074cd6721a02c8691f4b10705e Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Mon, 3 Mar 2025 14:24:16 +0100
Subject: [PATCH] Update setup.py

---
 setup.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/setup.py b/setup.py
index c35dadb..880c7a5 100644
--- a/setup.py
+++ b/setup.py
@@ -192,12 +192,12 @@ def build_extension(self, ext: CMakeExtension) -> None:
                                           build_ext=CMakeBuild)),
     zip_safe=False,
     url='https://github.com/ICAMS/python-ace',
-    install_requires=['numpy<=1.26.4',
+    install_requires=['numpy',
                       'ase',
-                      'pandas<=2.0',
+                      'pandas',
                       'ruamel.yaml',
                       'psutil',
-                      'scikit-learn<=1.4.2'
+                      'scikit-learn'
                       ],
     classifiers=[
         'Programming Language :: Python :: 3',
