From 053568cd536b9c13c3c62a0bed0c5905a383bd3d Mon Sep 17 00:00:00 2001
From: yuzie007 <yuji.ikeda.ac.jp@gmail.com>
Date: Wed, 12 Apr 2023 00:42:02 +0200
Subject: [PATCH] Modify `-DCMAKE_INSTALL_RPATH` for MacOS

---
 setup.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index fae64ed..11d6154 100644
--- a/setup.py
+++ b/setup.py
@@ -71,7 +71,10 @@ def build_extension(self, ext):
 
         # Pile all .so in one place and use $ORIGIN as RPATH
         cmake_args += ["-DCMAKE_BUILD_WITH_INSTALL_RPATH=TRUE"]
-        cmake_args += ["-DCMAKE_INSTALL_RPATH={}".format("$ORIGIN")]
+        if platform.system() == "Darwin":  # MacOS
+            cmake_args += ["-DCMAKE_INSTALL_RPATH={}".format("@loader_path")]
+        else:
+            cmake_args += ["-DCMAKE_INSTALL_RPATH={}".format("$ORIGIN")]
         cmake_args += ["-DBUILD_SHARED_LIBS=ON"]
         cmake_args += ["-DYAML_BUILD_SHARED_LIBS=ON"]
 
