From a66c310a9ab16759c25553df89d90eb2aa07dfeb Mon Sep 17 00:00:00 2001
From: Yury Lysogorskiy <yura.lysogorskii@gmail.com>
Date: Wed, 22 Nov 2023 16:49:17 +0100
Subject: [PATCH] fix yaml.safe_load

---
 bin/pacemaker          | 3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/bin/pacemaker b/bin/pacemaker
index a1e0f50..da559ac 100644
--- a/bin/pacemaker
+++ b/bin/pacemaker
@@ -169,7 +169,8 @@ def main(args):
     log.info("ace_evaluator   version: {}".format(get_ace_evaluator_version()))
     log.info("Loading {}... ".format(input_yaml_filename))
     with open(input_yaml_filename) as f:
-        args_yaml = yaml.safe_load(f)
+        yaml_loader = yaml.YAML(typ='safe', pure=True)
+        args_yaml = yaml_loader.load(f)
 
     assert isinstance(args_yaml, dict)
     if "cutoff" in args_yaml:
